kickstart/PXEGrub.erb

```diff
@@ -8,18 +8,63 @@ oses:
 -%>
 # This file was deployed via '<%= template_name %>' template
 
+<%
+  major = @host.operatingsystem.major.to_i
+  mac = @host.provision_interface.mac
+
+  # Tell Anaconda to perform network functions with boot interface
+  options = ["network",]
+  if mac
+    bootif = '00-' + mac.gsub(':', '-')
+    options.push("BOOTIF=#{bootif}")
+  end
+
+  # Tell Anaconda what to pass off to kickstart server
+  if (@host.operatingsystem.name.match(/Fedora/i) && major < 17) || major < 7
+    options.push("kssendmac", "ksdevice=bootif")
+  else
+    options.push("ks.device=bootif", "ks.sendmac", "inst.ks.sendmac")
+  end
```

Comment:
Can you shorten this to:

    options.push("ks.device=bootif", "kssendmac", "ks.sendmac", "inst.ks.sendmac")

The if statement is no longer necessary, it's better to provide all three possible variations of this Anaconda flag. Do this in all templates please.